source/code/plugins/patch_management_lib.rb

```diff
@@ -11,9 +12,13 @@ class LinuxUpdates
     @@prev_hash = ""
     @@delimiter = "_"
     @@force_send_last_upload = Time.now
+
     MAJOR_MINOR_VERSION_REGEX = /([^\.]+)\.([^\.]+).*/
     OMS_ADMIN_FILE = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
     SCX_RELEASE_FILE = "/etc/opt/microsoft/scx/conf/scx-release"
+    CURRENT_UPDATE_RUN_FILE = "/tmp/schedule_run.id"
```

Comment:
move it to be a filter config and set the default value to a file under /var/opt/microsoft/omsagent/state